src/plugins/tables/tables-en.hbs

```diff
@@ -972,193 +972,5 @@
-								<li class="checkbox"><label for="dt_location11"><input type="checkbox" id="dt_location11" name="dt_location" data-column="5" value="Quebec"/> Quebec</label></li>
-								<li class="checkbox"><label for="dt_location12"><input type="checkbox" id="dt_location12" name="dt_location" data-column="5" value="Yukon Territory"/> Yukon Territory</label></li>
-								<li class="checkbox"><label for="dt_location13"><input type="checkbox" id="dt_location13" name="dt_location" data-column="5" value="Prince Edward Island"/> Prince Edward Island</label></li>
-							</ul>
-						</fieldset>
-						<details>
-							<summary><h6>Advanced Options</h6></summary>
-							<div class="radio">
-								<label for="advanced_options1_1"><input type="radio" name="advanced_options1" id="advanced_options1_1" data-aopts='{"type": "any", "column": "5"}' checked />Any</label>
-							</div>
-							<div class="radio">
-								<label for="advanced_options1_2"><input type="radio" name="advanced_options1" id="advanced_options1_2" data-aopts='{"type": "both", "column": "5"}' />Both</label>
-							</div>
-							<div class="radio">
-								<label for="advanced_options1_3"><input type="radio" name="advanced_options1" id="advanced_options1_3" data-aopts='{"type": "either", "column": "5"}' />Either</label>
-							</div>
-							<div class="radio">
-								<label for="advanced_options1_4"><input type="radio" name="advanced_options1" id="advanced_options1_4" data-aopts='{"type": "and", "column": "5"}' />And</label>
-							</div>
-						</details>
-					</div>
-					<div class="form-group">
-						<button type="submit" class="btn btn-primary" aria-controls="dataset-filter">Filter</button>
-					</div>
-					<div class="form-group">
-						<button type="reset" class="btn btn-link btn-sm p-0">Reset to defaults</button>
-					</div>
-				</form>
-			</details>
-		</div>
-		<div class="col-md-9">
-			<table class="wb-tables table table-striped table-hover" id="dataset-filter" aria-live="polite" data-wb-tables='{
-			"bDeferRender": true,
-			"ajaxSource": "ajax/datasource.json",
-			"order": [5, "desc"],
-			"columns": [
-				{ "data": "TITLE", "className": "nws-tbl-ttl h4" },
-				{ "data": "PUBDATE", "className": "nws-tbl-date" },
-				{ "data": "DEPT", "className": "nws-tbl-dept" },
-				{ "data": "TYPE", "className": "nws-tbl-type" },
-				{ "data": "TEASER",  "className": "nws-tbl-desc" },
-				{ "data": "LOCATION",  "visible": false },
-				{ "data": "AUDIENCE",  "visible": false },
-				{ "data": "SUBJECT", "visible": false },
-				{ "data": "MINISTER", "visible": false }
-			]}'>
-				<thead>
-				<tr>
-					<th>Title</th>
-					<th>Publication date</th>
-					<th>Department</th>
-					<th>News Type</th>
-					<th>Summary</th>
-					<th>Location</th>
-					<th>For</th>
-					<th>Subject</th>
-					<th>Minister</th>
-				</tr>
-				</thead>
-			</table>
-		</div>
-	</div>
-
-
-
+	<p>View example <a href="../../demos/tables/tables-fltrs-en.html">here</a>.</p>
```

Comment:
Instead of this, can you add a table of content with anchor at the top of the page?

Kind of like fieldflow working example: https://wet-boew.github.io/themes-dist/GCWeb/demos/fieldflow/fieldflow-en.html